modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -290,16 +292,22 @@ public function getReservations(AlephPatron $patron) {
    * Create a reservation.
    *
    * @param \Drupal\aleph\Aleph\Entity\AlephPatron $patron
-   *    The Aleph patron.
+   *   The Aleph patron.
    *
    * @param \Drupal\aleph\Aleph\Entity\AlephRequest $request
-   *    The request information.
+   *   The request information.
+   *
+   * @param \Drupal\aleph\Aleph\Entity\AlephHoldGroup[] $holding_groups
+   *   The holding groups.
    *
    * @return \SimpleXMLElement
-   * @throws \RuntimeException
+   *   The SimpleXMLElement from the raw XML response.
+   *
+   * @throws AlephClientException
    */
-  public function createReservation(AlephPatron $patron, AlephRequest $request) {
+  public function createReservation(AlephPatron $patron, AlephRequest $request, $holding_groups) {
     $options = array();
+    $response = FALSE;
```

Comment:
Why do we need to set response to `FALSE`? It does not seem to be in line with the parameter declaration.